### PR TITLE
feat: Add lag tracker to log event loop lag and fail if lag too high

### PR DESCRIPTION
**Pull request by sommestad**
feat: Add lag tracker to log event loop lag and fail if lag too high

**TODO**
- [x] Make it possible to provide config values, or this won't work well in tests

### CODE REVIEW

File: src/lag-tracking/event-loop-lag-monitor.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import event_loop_lag from 'event-loop-lag';
+
+const REFRESH_LAG_INTERVAL = 1000;
+
+const api = {
+  currentEventLoopLag: event_loop_lag(REFRESH_LAG_INTERVAL),
```
Comment (dbrockman) [inline src/lag-tracking/event-loop-lag-monitor.js]: This lag tracker is created even if this feature of the module is not used

File: src/lag-tracking/lag-tracker.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+
+function create(bunyan_logger, { max_allowed_lag = 70 } = {}) {
+  const logger = bunyan_logger.child({
+    context: 'lag_tracker',
+  });
+
+  const config = logging_config.getDefaultConfig();
+  const logLag = createThrottledMaxLagLogger(logger, config.min_lag_to_be_logged, config.logging_interval_millis);
```
Comment (dbrockman) [inline src/lag-tracking/lag-tracker.js]: Doesn't it make more sense to create `event_loop_lag_monitor` here? Or will it become difficult to test then?

**Comment by dbrockman** [comment]
@sommestad I think it might be a bit broken to start the lag tracker when the module is loaded but I merged it anyway

File: src/lag-tracking/lag-tracker.js (hunk 2 above)
Comment (sommestad) [inline src/lag-tracking/lag-tracker.js]: The split was only to make it possible/easier to test

File: src/lag-tracking/event-loop-lag-monitor.js (hunk 1 above)
Comment (sommestad) [inline src/lag-tracking/event-loop-lag-monitor.js]: Hm, true. Might want to rethink that.

**Pull request merged**